### PR TITLE
Add heloufir-filament-kanban plugin

### PR DESCRIPTION
**Pull request by heloufir**

### CODE REVIEW

**Comment by heloufir** [comment]
For information the author profile is added in a previous PR #221

File: content/plugins/heloufir-filament-kanban.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+anystack_id: 9a833198-82cc-42db-836d-a62aaf9e0a15
+name: Filament Kanban
+slug: heloufir-filament-kanban
+author_slug: heloufir
+categories: [filament]
+description: Integrates Kanban into Filament projects, simplifying task management, progress tracking, and team collaboration, enhancing productivity and organization.
+docs_url: https://gist.github.com/heloufir/05a86302f871fa1a81ac05d693d9c9d9
```
Comment (awcodes) [inline content/plugins/heloufir-filament-kanban.md:8]: Please provide the docs url to the raw content of the md.

**Review by awcodes — CHANGES_REQUESTED** [review]

File: content/plugins/heloufir-filament-kanban.md (hunk 1 above)
Comment (heloufir) [reply content/plugins/heloufir-filament-kanban.md:8]: @awcodes done you can check

Comment (heloufir) [reply content/plugins/heloufir-filament-kanban.md:8]: Can you please give more info? the docs url is already mentioned as the Filament docs says

**Review by awcodes — COMMENTED** [review]
From the Contributing notes on the Readme.

The docs_url should be a URL to a public, raw Markdown file of your plugin. You can leave this blank if your documentation does not live in a raw Markdown file, but please ensure that you have filled in a url instead, where we can redirect users who are looking for the documentation. If you have content in your README that you do not want to be displayed on the website, please add a .filament-hidden class to the element. This is especially useful for banner images.

File: content/plugins/heloufir-filament-kanban.md (hunk 1 above)
Comment (heloufir) [reply content/plugins/heloufir-filament-kanban.md]: @awcodes my bad, I understand now thanks :)
I added the raw url of the docs

**Comment by awcodes** [comment]
You also need to provide an author file since this is your first v3 plugin submission. Please.

**Comment by heloufir** [comment]
> You also need to provide an author file since this is your first v3 plugin submission. Please.

I

> For information the author profile is added in a previous PR #221

@awcodes I mentioned here that I have created an old PR with another plugin + my author profile, do I need to add my author profile here too?

**Comment by awcodes** [comment]
Just saw your other plugin. Sorry. you're good. Dan will just need to merge that one first. cheers.

**Review by awcodes — APPROVED** [review]

**Review by awcodes — CHANGES_REQUESTED** [review]
Please remove 'filament' from all slugs and filenames. It is ok in your repo name, but should not be part of anything else.

**Comment by heloufir** [comment]
@awcodes done for the slug and filenames, I just removed `filament`

**Review by awcodes — APPROVED** [review]

**Review by saade — CHANGES_REQUESTED** [review]
Hey! Your plugin was submitted to a nonexistent category, please choose one of the [available categories](https://github.com/filamentphp/filamentphp.com/tree/main/content/plugin_categories) and let us know when you're ready.

**Comment by heloufir** [comment]
> Hey! Your plugin was submitted to a nonexistent category, please choose one of the [available categories](https://github.com/filamentphp/filamentphp.com/tree/main/content/plugin_categories) and let us know when you're ready.

Done

**Review by saade — APPROVED** [review]

**Comment by saade** [comment]
#221 needs to be merged before this PR since it contains the author profile.

**Pull request merged**